### DISCUSSION TITLE
Partial static rendering would be more useful

### DISCUSSION THREAD

**Discussion by chhornponleu** [expo/router#791] [Ideas]
Some dynamic routes have too much data to generate static routes. It would be good to provide a way to configure which path to do static, or either which paths to exclude.

**Comment by cseelus** [comment]
Yes, it would be great if Expo Router enabled some routes to be marked as static, for example via the `_layout` files, for exactly the reason you stated.

**Reply by murraybauer** [reply]
Agree, having a route as either static or dynamic is a less flexible/older model, and no longer relevant with PPR (available in Next.js 15 canary versions) that combines both automatically.

Vercel did a great explainer video here: https://www.youtube.com/watch?v=MTcPrTIBkpA
